src/main/java/com/exasol/releasedroid/usecases/PropertyReaderImpl.java

```diff
@@ -21,6 +22,18 @@
      */
     public PropertyReaderImpl(final String pathToPropertyFile) {
         this.pathToPropertyFile = pathToPropertyFile;
+        this.consoleReader = new ConsoleReaderImpl();
```

Comment:
```suggestion
        this(pathToPropertyFile, new ConsoleReaderImpl());
```
Call other constructor